src/test/java/com/searchservice/app/domain/dto/logger/CorrelationIDTest.java

```diff
@@ -0,0 +1,18 @@
+package com.searchservice.app.domain.dto.logger;
+
+import org.junit.jupiter.api.Test;
+
+public class CorrelationIDTest {
+  
+	CorrelationID c1=new CorrelationID();
+	CorrelationID c2=new CorrelationID();
+	@Test 
+	void testCorrelationID()
+	{
+		c1.hashCode();
```

Comment:
This is not the way to create JUnit tests, there must be assertions for individual blocks or parameters, but may be we can ignore the changes here, since it's just a DTO